### ISSUE TITLE
psutil installation problem on MacOs (Catalina)

### ISSUE THREAD

**Issue by rizac** [gem/oq-engine#6112]
Hi, 

I recently found installation issues (the usual way: `pip install openquake.engine`) on macOS (Catalina): the problem seems to be due to psutil (see this [thread](https://github.com/giampaolo/psutil/issues/1632) for details). I guess it might be simply necessary to update psutil to more recent versions. For instance, by manually clone the repo and  run `pip install .` after changing the following line `setup.py`:

```python
install_requires = [
    ...
    'psutil >=2.0, <5.8',  # <-- changed from 5.7 to 5.8
    ...
]
```
the installation was successful

**Comment by raoanirudh** [comment]
Just tried installing the engine on OS Catalina, it completes successfully without issues.


```sh
python3.8 -m venv test
source test/bin/activate
```

```sh
pip install --upgrade pip setuptools
Collecting pip
  Using cached pip-20.2.3-py2.py3-none-any.whl (1.5 MB)
Collecting setuptools
  Using cached setuptools-50.3.0-py3-none-any.whl (785 kB)
Installing collected packages: pip, setuptools
  Attempting uninstall: pip
    Found existing installation: pip 20.1.1
    Uninstalling pip-20.1.1:
      Successfully uninstalled pip-20.1.1
  Attempting uninstall: setuptools
    Found existing installation: setuptools 47.1.0
    Uninstalling setuptools-47.1.0:
      Successfully uninstalled setuptools-47.1.0
Successfully installed pip-20.2.3 setuptools-50.3.0
```

```sh
pip install openquake.engine
Collecting openquake.engine
  Downloading openquake.engine-3.10.0-py3-none-any.whl (31.7 MB)
     |████████████████████████████████| 31.7 MB 6.6 MB/s
Collecting scipy<1.5,>=1.3
  Using cached scipy-1.4.1-cp38-cp38-macosx_10_9_x86_64.whl (28.8 MB)
Collecting pandas<1.1,>=0.25
  Downloading pandas-1.0.5-cp38-cp38-macosx_10_9_x86_64.whl (10.2 MB)
     |████████████████████████████████| 10.2 MB 32.4 MB/s
Collecting requests<2.24,>=2.20
  Downloading requests-2.23.0-py2.py3-none-any.whl (58 kB)
     |████████████████████████████████| 58 kB 21.0 MB/s
Collecting numpy<1.19,>=1.16
  Downloading numpy-1.18.5-cp38-cp38-macosx_10_9_x86_64.whl (15.1 MB)
     |████████████████████████████████| 15.1 MB 4.9 MB/s
Collecting matplotlib<3.2,>=1.5
  Using cached matplotlib-3.1.3-cp38-cp38-macosx_10_9_x86_64.whl (13.2 MB)
Collecting decorator>=4.3
  Using cached decorator-4.4.2-py2.py3-none-any.whl (9.2 kB)
Collecting shapely<1.8,>=1.3
  Using cached Shapely-1.7.1-cp38-cp38-macosx_10_9_x86_64.whl (1.0 MB)
Collecting pyshp==1.2.3
  Downloading pyshp-1.2.3.tar.gz (28 kB)
Collecting h5py<2.11,>=2.9
  Using cached h5py-2.10.0-cp38-cp38-macosx_10_9_x86_64.whl (3.0 MB)
Collecting docutils<0.15,>=0.11
  Downloading docutils-0.14-py3-none-any.whl (543 kB)
     |████████████████████████████████| 543 kB 32.0 MB/s
Requirement already satisfied: setuptools in ./test/lib/python3.8/site-packages (from openquake.engine) (50.3.0)
Collecting pyzmq<20.0
  Using cached pyzmq-19.0.2-cp38-cp38-macosx_10_9_x86_64.whl (806 kB)
Collecting django<2.3,>=1.11
  Downloading Django-2.2.16-py3-none-any.whl (7.5 MB)
     |████████████████████████████████| 7.5 MB 39.3 MB/s
Collecting toml
  Using cached toml-0.10.1-py2.py3-none-any.whl (19 kB)
Collecting psutil<5.7,>=2.0
  Downloading psutil-5.6.7.tar.gz (448 kB)
     |████████████████████████████████| 448 kB 44.7 MB/s
Collecting python-dateutil>=2.6.1
  Using cached python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
Collecting pytz>=2017.2
  Using cached pytz-2020.1-py2.py3-none-any.whl (510 kB)
Collecting chardet<4,>=3.0.2
  Using cached chardet-3.0.4-py2.py3-none-any.whl (133 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2020.6.20-py2.py3-none-any.whl (156 kB)
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1
  Using cached urllib3-1.25.10-py2.py3-none-any.whl (127 kB)
Collecting idna<3,>=2.5
  Using cached idna-2.10-py2.py3-none-any.whl (58 kB)
Collecting cycler>=0.10
  Using cached cycler-0.10.0-py2.py3-none-any.whl (6.5 kB)
Collecting pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1
  Using cached pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)
Collecting kiwisolver>=1.0.1
  Using cached kiwisolver-1.2.0-cp38-cp38-macosx_10_9_x86_64.whl (60 kB)
Collecting six
  Using cached six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting sqlparse>=0.2.2
  Downloading sqlparse-0.3.1-py2.py3-none-any.whl (40 kB)
     |████████████████████████████████| 40 kB 20.3 MB/s
Using legacy 'setup.py install' for pyshp, since package 'wheel' is not installed.
Using legacy 'setup.py install' for psutil, since package 'wheel' is not installed.
Installing collected packages: numpy, scipy, six, python-dateutil, pytz, pandas, chardet, certifi, urllib3, idna, requests, cycler, pyparsing, kiwisolver, matplotlib, decorator, shapely, pyshp, h5py, docutils, pyzmq, sqlparse, django, toml, psutil, openquake.engine
    Running setup.py install for pyshp ... done
    Running setup.py install for psutil ... done
Successfully installed certifi-2020.6.20 chardet-3.0.4 cycler-0.10.0 decorator-4.4.2 django-2.2.16 docutils-0.14 h5py-2.10.0 idna-2.10 kiwisolver-1.2.0 matplotlib-3.1.3 numpy-1.18.5 openquake.engine-3.10.0 pandas-1.0.5 psutil-5.6.7 pyparsing-2.4.7 pyshp-1.2.3 python-dateutil-2.8.1 pytz-2020.1 pyzmq-19.0.2 requests-2.23.0 scipy-1.4.1 shapely-1.7.1 six-1.15.0 sqlparse-0.3.1 toml-0.10.1 urllib3-1.25.10
```

**Comment by rizac** [comment]
Weirdly, after installing psutil 5.7 the way I did, I tried again on a new venv (basically, as in the example above) and it works.
So, just FYI, the problem might be reproducible only on the first install **without using** a cached version, as it was in my first attempt (I am re-installing stuff on a brand new mac-os). .
Whatever, it works. I guess we can close the issue. Sorry for bothering and thank you very much
